pkgs/development/python-modules/databricks-connect/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, isPy37, pkgs, buildPythonPackage, fetchPypi, six, py4j }:
+
+buildPythonPackage rec {
+  pname = "databricks-connect";
+  version = "7.1.0";
+  sha256 = "996a9d0f271f6c7edbd2d85b2efb6ff4e58d15222e80f87ca17fdbf224e17056";
```

Comment:
The pname and version are fine here, but the sha256 would be more normal in the args to fetchPypi

Reply:
Have now moved sha256 into fetchPypi